mmeds/resources/summary_code.txt

```diff
@@ -58,12 +60,12 @@ library(RColorBrewer)
 library(GGally)
 library(ggrepel)
 
-# Create custom color palette
+# Create custom color palette from brewer "Set1"
 myColors <- brewer.pal(8, "Set1")
-if (length(myColors) > length(unique(all_colors))) {
+if (length(myColors) >= length(unique(all_colors))) {
     allColors <- myColors[0 : length(unique(all_colors))]
-}
-else {
+} 
+else { # Greater than 8 colors, extrapolate to get more
```

Comment:
I think it's better to have comments before code blocks, explaining what they're for. 
Also, not sure what pep8 has to say about comments and code on the same line, but I doubt they're in favor of it.